tests/unit/regression/lars_unittest.cc

```diff
@@ -396,7 +395,7 @@ TEST(LeastAngleRegression, cholesky_insert)
 	
 	MatrixXd XX=mat_old.transpose()*mat_old;
 	// Compute matrix R which has to be updated
-	SGMatrix<float64_t> R_old=linalg::cholesky(XX, false);
+	SGMatrix<float64_t> R_old=linalg::cholesky_factor(SGMatrix<float64_t>(XX), false);
```

Comment:
in lars, we also update the cholesky.
This is a method `linalg` should also have, put it on the list :)